sys/include/net/ieee802154/radio.h

```diff
@@ -621,6 +621,7 @@ struct ieee802154_radio_ops {
      * function should return -EINVAL
      *
      * @pre the device is on
+     * @pre the transceiver state is @ref IEEE802154_TRX_STATE_TRX_OFF
```

Comment:
Btw: Care to sneak in some Doxygen fixes?

```diff
-     * @return 0 on success
-     * @return -EINVAL if the configuration is not valid for the device.
-     * @return negative errno on error
+     * @retval 0        success
+     * @retval -EINVAL  the configuration is not valid for the device.
+     * @retval <0       error, return value is negative errno indicating the cause
```

Reply:
done!